packages/@react-spectrum/s2/src/Modal.tsx

```diff
@@ -141,23 +115,22 @@ export const Modal = forwardRef(function Modal(props: ModalProps, ref: DOMRef<HT
             value: 'layer-2'
           },
           backgroundColor: '--s2-container-bg',
-          animation: {
-            isEntering: fadeAndSlide,
-            isExiting: fade
+          opacity: {
+            isEntering: 0,
+            isExiting: 0
+          },
+          translateY: {
+            isEntering: 20
           },
-          animationDuration: {
-            isEntering: 250,
+          transition: '[opacity, translate]',
+          transitionDuration: {
+            default: 250,
```

Comment:
A bit weird that this is not symmetric with `isExiting`. `isEntering` won't work because it is only applied for a single frame.